README.md

```diff
@@ -72,14 +72,16 @@
 
 - **[Creating a data model]**
 - **[Process data in model]**
+- **[Added Validations in model]**
+- **[Added Unit Tests in model]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo]()
+- [Live Demo](CommingSoon)
```

Comment:
- [ ] @Metaverse-Nabeel Great work on your documentation, however I would suggest you remove this section or add a link to your  live demo app

Reply:
Hello @danmainah 

Thank you for your feedback. I have made the required change.

Regards